Packs/Cisco-umbrella-cloud-security/Integrations/Cisco-umbrella-cloud-security/Cisco-umbrella-cloud-security.py

```diff
@@ -53,6 +53,20 @@ def add_domain(client: Client, **args) -> str:
     return f'Domain {", ".join(destinations)} successfully added to list {r["data"]["name"]}'
 
 
+def remove_domain(client: Client, **args) -> str:
+    # https://docs.umbrella.com/umbrella-api/reference#delete_v1-organizations-organizationid-destinationlists-destinationlistid-destinations-remove
+    organizationId = args.get('orgId')
+    destinationListId = args.get('destId')
+    uri = f'{organizationId}/destinationlists/{destinationListId}/destinations/remove'
```

Comment:
Will this uri be valid even if no organizationId is provided?

Reply:
Yes, it is taken from instance config and there it is required. So it's covered here.

Reply:
Looks like the orgID is not mandatory in the yml 
![image](https://user-images.githubusercontent.com/50294648/109796516-ed7b2380-7c20-11eb-9e7e-ebaa8f654475.png)

Reply:
@BlazejG do you want to make it mandatory or do you want to merge like that?

Reply:
Merge like that